lib/torque/animator.js

```diff
@@ -118,9 +118,15 @@ var cancelAnimationFrame = global.cancelAnimationFrame
       this._t0 = t1;
       this._time += delta;
       if(this.step() >= this.options.steps) {
-        this._time = 0;
         if(!this.options.loop){
-          this.stop();
+          // set time to max time
+          this.time(this.options.animationDuration);
+          // go to final frame (callback to nothing)
+          requestAnimationFrame(function(){});
```

Comment:
Without it, the animation might stop on a frame that is not the final. The animation does not call every single step - for this reason, I couldn't intercept the last frame in the client library. In my case, I'm using the SQL api to call the table and display counts, so if we stop on, say 506 of 512, the counts for that timestamp wont be the final.